astro/src/content/docs/apis/_user-request-body.mdx

```diff
@@ -29,9 +30,13 @@ You must specify either the **email** or the **username** or both for the User.
   }
 
   <APIField name="skipVerification" type="Boolean" optional defaults="false">
-    Indicates to FusionAuth that it should skip email verification even if it is enabled. This is useful for creating admin or internal User accounts.
+    Indicates to FusionAuth that it should skip email and phone verification even if it is enabled. This is useful for creating admin or internal User accounts.
 
-    Setting this to `true` will set the <InlineField>user.verified</InlineField> field to `true` as well.
+    Setting this to `true` will, for all identities created or updated in this request:
+
+    * Set <InlineField>user.identities\[x].verified</InlineField> field to `false`.
```

Comment:
We tend to not use an ending period on bullet list items with single segment fragments.